get-secretmanager-secrets/.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules/
```

Comment:
Please copy the .gitignore from the setup-gcloud action: https://github.com/GoogleCloudPlatform/github-actions/blob/master/setup-gcloud/.gitignore